### PR TITLE
Update Timeout.js

### PR DESCRIPTION
**Pull request by miroxik74**

### CODE REVIEW

**Comment by miroxik74** [comment]
seems like new errors occurred and i don't know why, i'll try...

**Comment by miroxik74** [comment]
Solution has been found!
Please note that this mod requires `BetterMods`, and no, it's not about `AAA_BetterMods`.
Once you have downloaded this file, save it as `BetterMods.js` and add it to `mods` folder, then Timeout mod will work
[BetterMods.txt](https://github.com/discord-bot-studio/mods/files/13161686/BetterMods.txt)

**Comment by miroxik74** [comment]
Hey, unfortunately this mod is a little problematic, so here are requirements you must do:
1. in cmd, run: `npm install discord.js@13.9` 
2. this mod requires `BetterMods`, which means you have to download this file and save it as `BetterMods.js`, then add it into `mods` folder.
[BetterMods.txt](https://github.com/user-attachments/files/15569796/BetterMods.txt)

3. Do not leave the field "User to Timeout" empty, for unknown reason to me it doesn't work. Instead, store variable as number, then put this variable as user to timeout. 
In conclusion, it works but only with user's id, i couldn't find the solution for mention..
